src/main/java/GeneralService/OnlineServicesCommunicationService.java

```diff
@@ -0,0 +1,27 @@
+package GeneralService;
+
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+
+@Service
+public class OnlineServicesCommunicationService {
+
+    @Autowired
+    private MqttClient mqttClient;
+
+    public void sendRequest(Request request) throws MqttException {
+        if (!mqttClient.isConnected())
+            mqttClient.connect();
+
+        boolean isValidRequestType = Arrays.asList(RequestType.values()).contains(request.getRequestType());
```

Comment:
you're not using this isValidRequestType variable for anything else besides the below if, delete the variable and put the check in the if directly